apps/voting/contracts/Voting.sol

```diff
@@ -263,7 +295,8 @@ contract Voting is IForwarder, AragonApp {
             uint256 yea,
             uint256 nay,
             uint256 votingPower,
-            bytes script
+            bytes script,
+            bool openForObjection
```

Comment:
The parameter is not documented above

Reply:
@param _voteId Vote identifier
@return if the vote open for both support and objection
...
@return if the vote open for objection